bin/pathway_analysis.R

```diff
@@ -82,6 +83,12 @@ if(is.null(opt$species)){
   stop("Species name is unknown, check for typos or contact responsible person to add your species.")
 }
 
+# Default to 1 for the nb of genes DE in a pathway
+if(!is.na(as.integer(opt$min_DEG_pathway))) {
+  min_DEG_pathway <- opt$min_DEG_pathway
+} else {
+  min_DEG_pathway = 1
+}
```

Comment:
```suggestion
 min_DEG_pathway <- as.integer(opt$min_DEG_pathway)
```

Reply:
You can set the default when you use the `make_option` statement (see above), so you do not need to check it here again